dipy/workflows/analysis.py

```diff
@@ -0,0 +1,332 @@
+from __future__ import division, print_function, absolute_import
```

Comment:
analysis.py cool! We should have something similar inside the DIPY API too. Something to talk during the workshop.